packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -18,9 +19,14 @@ import { ICacheEntry, LocalPersistentCacheAdapter } from "./odspCache";
 export class EpochTracker {
     private _fluidEpoch: string | undefined;
     private _hashedDocumentId: string | undefined;
+    private readonly rateLimiter: Bottleneck;
     constructor(
         private readonly persistedCache: LocalPersistentCacheAdapter,
-        private readonly logger?: ITelemetryLogger) {
+        private readonly logger?: ITelemetryLogger,
+    ) {
+        this.rateLimiter = new Bottleneck({
+            maxConcurrent: 12, // Limits the max number of concurrent requests to 12.
```

Comment:
This is likely Ok, given default limit of 6 sockets per domain.
That said, when we get http/2 that would be too low.
I wonder if we want to go here with higher limit we can get away before browser fails.
Assuming that we understand why it fails, and limits across all supported browsers.

Otherwise it's Ok, I guess.

Reply:
We are going for 24 currently. If we say delays because of that then we can try to increase the max requests,